processing/src/main/java/org/apache/carbondata/processing/util/CarbonDataProcessorUtil.java

```diff
@@ -401,6 +420,33 @@ public static boolean isHeaderValid(String tableName, String[] csvHeader,
     return noDicSortColMapping;
   }
 
+  /**
+   * get mapping based on data fields order
+   *
+   * @param dataFields
+   * @return
+   */
+  public static boolean[] getNoDictSortColMappingAsDataFieldOrder(DataField[] dataFields) {
+    List<Boolean> noDicSortColMap = new ArrayList<>();
+    for (DataField dataField : dataFields) {
```

Comment:
same as above

Reply:
all these functions are replicated from the same file. Whole sort step has dictionary and no-dictionary mapping. 
I do agree that we don't have dictionary. But cannot change all the current sort step variables and logic in this PR.